inst/templates/mapping_template_NAVIGATE.csv

```diff
@@ -998,17 +998,23 @@ idx;Tier;Category;Variable;Unit;piam_variable;piam_unit;piam_factor;internal_com
 951;1;energy (final);Final Energy|Transportation|Bus|Hybrid|Electricity;EJ/yr;;;;;;final energy consumption of liquid fuels in the transportation sector by Hybrid Bus;
 952;1;energy (final);Final Energy|Transportation|Bus|Hybrid|Liquids;EJ/yr;;;;;;final energy consumption of liquid fuels in the transportation sector by Hybrid Bus;
 953;1;energy (final);Final Energy|Transportation|Bus|Hybrid|Gases;EJ/yr;;;;;;final energy consumption of gases in the transportation sector by Hybrid Bus;
+954;1;energy (final);Final Energy|Transportation|Passenger;EJ/yr;FE|Transport|Pass|Pass|w/o Bunkers;EJ/yr;;;;final energy consumed for passenger transportation;x
```

Comment:
not sure what is going on here, these seem to be bigger changes…

Reply:
Before there was a bug in the NAVIGATE template: Final Energy|Transportation|Passenger was mapped to FE|Transport|Liquids|Oil

I changed this to FE|Transport|Pass|Pass|w/o Bunkers